### PR TITLE
Platform halt

### PR DESCRIPTION
**Pull request by xjcaa**

### CODE REVIEW

File: src/market.ts [hunk 1]
```diff
@@ -599,7 +598,5 @@ export class Market {
         await processTransaction(Exchange.provider, tx);
       })
     );
-
-    await sleep(1000);
```
Comment (xjcaa) [inline src/market.ts:603]: @kelvinlau20100 do you remember why we had this sleep lol

File: src/exchange.ts [hunk 2]
```diff
@@ -1016,6 +940,90 @@ insuranceVaultLiquidationPercentage=${params.insuranceVaultLiquidationPercentage
...
+    this.assertHalted();
+    await Promise.all(
+      this._markets.markets.map(async (market) => {
+        await market.cancelAllOrdersHalted();
+      })
+    );
+  }
+
+  public async cleanZetaMarketsHalted() {
+    this.assertHalted();
```
Comment (0xtristan) [inline src/exchange.ts:995]: Is this not already asserted on the rust side?

**Review by 0xtristan — APPROVED** [review]
Yeet, looks great

File: src/exchange.ts (hunk 2 above)
Comment (xjcaa) [reply src/exchange.ts:995]: Rather it fail in the client too.

**Pull request merged**